webpack/assets/javascripts/react_app/components/DeleteMessageDialog/DeleteConfirmationDialog.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MessageDialog, Icon, Button, Spinner } from 'patternfly-react';
+import { translate as __ } from '../../common/I18n';
+import { noop } from '../../common/helpers';
+import './deleteConfirmationDialog.scss';
+
+const DeleteConfirmationDialog = ({
+  controller,
+  id,
+  name,
+  show,
+  processing,
+  deleteItem,
+  closeDialog,
+  dialogTitle,
+  cancelText,
+  dialogContent,
+}) => (
+  <MessageDialog
+    show={show}
+    onHide={closeDialog}
+    footer={
+      <React.Fragment>
+        <Button bsStyle="default" onClick={closeDialog} disabled={processing}>
+          {cancelText}
+        </Button>
+        <Button
+          bsStyle="danger"
+          disabled={processing}
+          onClick={() => deleteItem()}
+          href={`/${controller}/${id}-${name}`}
+          data-method="delete"
+        >
+          <Spinner loading={processing} inline size="xs" />
+          {__(`Delete ${name}`)}
+        </Button>
+      </React.Fragment>
+    }
+    title={dialogTitle}
+    primaryContent={
+      <span className="lead">
+        {__(`Are you sure you want to delete ${name}`)}
+      </span>
+    }
+    secondaryContent={dialogContent}
+    icon={<Icon id="delete-modal-icon" type="pf" name="error-circle-o" />}
+    // TODO(boaz): Fix these on Patternfly side.
+    //             The problem is primaryAction and primaryActionButtonContent are
+    //             mandatory but if we use footer they're not necessary.
+    //             Read more: https://github.com/patternfly/patternfly-react/issues/1122
```

Comment:
seems like this was resolved by https://github.com/patternfly/patternfly-react/pull/1134 ?

Reply:
Thanks for noticing. Will update.